### DISCUSSION TITLE
[CONTRACTS] - Check Out LUSD!

### DISCUSSION THREAD

**Discussion by 0x4007** [pavlovcik/uad-ui#132] [Ideas]
https://twitter.com/LiquityProtocol/status/1349051458793123842 @hpiyankov

**Comment by hpiyankov** [comment]
I watched the video and read the documents.

What they do is cool and well thought out. That being said:

- As discussed their potential growth is limited by the fact that they only take ETH
- The way their ETH to LUSD redemptions work is questionable at best (e.g. I would not go long through them for this reason alone)
- They have not yet survived a major correction which means that they are working "in theory" but are not battle-tested
- I do not believe that we are at the level of sophistication for any coin that it does not need software updates and adjustments
- their price is actually not that stable - https://coinmarketcap.com/currencies/liquity-usd/

Long story short, I would not prefer them to USDC/USDT or DAI just yet.

**Reply by 0x4007** [reply]
> I watched the video and read the documents.
> 
> What they do is cool and well thought out. That being said:
> 
> * As discussed their potential growth is limited by the fact that they only take ETH

We can be layer 2 LUSD... ubLiquity 😀

> * The way their ETH to LUSD redemptions work is questionable at best (e.g. I would not go long through them for this reason alone)

I think it's only a risk to consider if you have a trove open. I am suggesting that we do not open a trove, and instead only hold LUSD.

> * They have not yet survived a major correction which means that they are working "in theory" but are not battle-tested
> * I do not believe that we are at the level of sophistication for any coin that it does not need software updates and adjustments

My gut feeling is good about LUSD - perhaps its simplicity? 

> * their price is actually not that stable - https://coinmarketcap.com/currencies/liquity-usd/

It's a young market, DAI was exactly the same when it was early on.

> Long story short, I would not prefer them to USDC/USDT or DAI just yet.

ser, pls.

**Reply by hpiyankov** [reply]
I am not saying "no", I am saying that I would not prefer it just yet. If you would like to switch, then it is "good enough". I am just sceptical at this point in time.

**Comment by zgorizzo69** [comment]
interesting can you elaborate on the redemption mechanism ?

**Comment by hpiyankov** [comment]
You go and deposit ETH in order to open a position. Later on, you can get your ETH back when you give back the LUSD... however

If someone comes to the platform and wants to redeem LUSD for ETH (without having an open position), the protocol starts liqudating existing positions (from the lowest LTV one up) in order to supply the ETH.

In a nutshell - you can be liquidated without ever reaching the liquidation ratio.

**Comment by Zequez** [comment]
I dig the decentralized independent frontend that act as independent marketing channels and get rewarded.

**Comment by bjnpck** [comment]
> As discussed their potential growth is limited by the fact that they only take ETH

We aimed for decentralization first, hence the just ETH decision. Would not say it limits growth.

> They have not yet survived a major correction which means that they are working "in theory" but are not battle-tested

May 19th drop was pretty bad, the protocol work exactly as intended, here our recap: https://medium.com/liquity/how-liquity-handled-its-first-big-stress-test-160f20d5b18f

> their price is actually not that stable

CMC is not that great of a price source, we set up this: https://dune.xyz/dani/LUSD-3CRV
But yes, there are more stable coins out there, we did not build this to be the absolutely most stable coin.

> If someone comes to the platform and wants to redeem LUSD for ETH (without having an open position)

Why would someone go and buy/swap LUSD and then go redeem for ETH? You go directly into ETH.

> you can be liquidated without ever reaching the liquidation ratio

This is not a liquidation but a redemption, the difference is that with a redemption you do not incur a loss -> lost debt = lost ETH. 
I was redeemed mid-May at $3.300, bought back ETH later at $2.400. Of course, it can go also against you, but the point is that at the moment of redemption there is no loss.

You also need to consider the redemption fee, which grows exponentially based on the amount you want to redeem. So to make a redemption economically sound LUSD must be enough under peg to accommodate for the fee as well (minimum is 0.5%, redeeming $50mil is 5% and $500m is 70%). 

If you check our redemption history, you will see the volume is not too big.

**Comment by 0x4007** [comment]
Would it be irresponsible to fork Liquity to create liquid staking derivatives? e.g. use uAD-3CRV LP tokens as collateral to mint new uAD or uBONDs?
